### PR TITLE
Fixed tutorial 'background.gif' reference.

### PR DESCRIPTION
**Pull request by adamchainz**
* ~'asgi.py' added out of order in 3930ec1bf275d17f1c36cd38b38114177b1d1565~
* 'background.gif' reference missed in 76fda7729e4cdfec715cd92b2c80d851797b05f7

### CODE REVIEW

File: docs/intro/tutorial01.txt [hunk 1]
```diff
@@ -84,9 +84,9 @@ Let's look at what :djadmin:`startproject` created:
         manage.py
         mysite/
             __init__.py
+            asgi.py
```
Comment (felixxm) [inline docs/intro/tutorial01.txt:87]: Now the file descriptions in the `These files are:` are not in the same order. I wanted to add `asgi.py` next to the `wsgi.py` to make it easier to read for newcomers.

Comment (nessita) [reply docs/intro/tutorial01.txt:87]: I agree that this change feels unnecessary, and that if we do merge it for whatever reason, the following section needs to be adjusted to use the same order as the filesystem list:
image

File: docs/intro/reusable-apps.txt [hunk 2]
```diff
@@ -76,7 +76,7 @@ After the previous tutorials, our project should look like this:
             static/
                 polls/
                     images/
-                        background.gif
+                        background.png
```
Comment (nessita) [inline docs/intro/reusable-apps.txt:79]: I agree this needs fixing to match the tutorial06 section. Shal we split it in a separated PR?

File: docs/intro/tutorial01.txt (hunk 1 above)
Comment (adamchainz) [reply docs/intro/tutorial01.txt:87]: Okay, fair enough, I will revert this bit.

**Comment by adamchainz** [comment]
This PR now only covers the `background.gif` change.

**Comment by felixxm** [comment]
@adamchainz Thanks :+1:

**Pull request merged**